controllers/user.js

```diff
@@ -101,10 +105,20 @@ async function getUnsubscribe(req, res) {
     throw new FluentError("error-not-subscribed");
   }
 
+  let headline = req.fluentFormat("unsub-headline");
+  let subhead = req.fluentFormat("unsub-blurb");
+
+  if (AppConstants.FXA_ENABLED) {
+    const fxaUrl = new URL(AppConstants.OAUTH_PROFILE_URI).origin;
+    const manageFxaLink = `<a class="manage-fxa" href="${fxaUrl}" target="_blank" rel="noopener"><span>${req.fluentFormat("manage-fxa")}</span>`;
```

Comment:
Does this need a `</a>`?

Also, what's the purpose of the `<span>` wrapper? Are we using that in CSS or something?

Reply:
Definitely doesn't need the span, definitely does need the `</a>`.

Reply:
I think we should move this `FXA_ENABLED` check into the view and use the `getFxaUrl` helper for the `href`.

Reply:
I.e., move all this logic into the `.hbs`